tools/sslsniff.py

```diff
@@ -211,7 +215,7 @@ def print_event(cpu, data, size, rw, evt):
     fmt = "%-12s %-18.9f %-16s %-6d %-6d\n%s\n%s\n%s\n\n"
     print(fmt % (rw, time_s, event.comm.decode('utf-8', 'replace'),
                  event.pid, event.len, s_mark,
-                 event.v0.decode('utf-8', 'replace'), e_mark))
+                 textwrap.fill(binascii.hexlify(event.v0).decode('utf-8', 'replace'),width=32) if args.hexdump else event.v0.decode('utf-8', 'replace'), e_mark))
```

Comment:
the line is too long, could you wrap into multiple lines? It is okay to use a temporary variable to avoid too long expression.